tls/tls_cli.c

```diff
@@ -1946,7 +1947,7 @@ int ttls_handshake_client_step(TlsCtx *ssl)
 		break;
 
 	   default:
-		   T_DBG("invalid state %d\n", ssl->state);
+		   T_WARN("invalid state %d\n", tls_state_to_str(ssl->state));
 		   return(TTLS_ERR_BAD_INPUT_DATA);
    }
 
```

Comment:
The file isn't used at all at the moment, but I think the changes are still good